src/App.css

```diff
@@ -112,6 +112,11 @@ h1 {
   line-height: 1;
 }
 
+.search-box {
+  display: unset;
+  width: unset;
```

Comment:
Totally up to you as to whether you want to correct this now or not (because we'll have time during the last couple of weeks of the cohort to style the app, etc), but I think the width can be increased here on the search box to be able to see the entire placeholder message :)

![Screenshot (104)](https://user-images.githubusercontent.com/45923677/117338379-c817c980-ae9e-11eb-8e05-643111b87e7f.png)

Reply:
Thank you very much, Alex! I'll make the adjustment.